.github/workflows/cifuzz.yaml

```diff
@@ -0,0 +1,24 @@
+name: CIFuzz
+on: [pull_request]
```

Comment:
Is this limitation on purpose, or do we also want to e.g. test branches? Also: do we want to test _all_ pull requests, or just the ones targeting specific branches?

Reply:
The goal for this workflow is just to ensure the fuzz targets are operational, and to identify any low-hanging fuzz (within 60s of execution). 

Assuming that _all_ changes to `main` are done via PRs, I think we want to target pull requests against `main` only. I will update that now.